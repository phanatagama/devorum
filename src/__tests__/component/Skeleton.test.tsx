import { render, screen } from '@testing-library/react';

import Skeleton from '@/components/Skeleton';
/**
 * Test scenario for Skeleton component
 *
 * - Should display element with class animate-shimmer when render is correctly
 */
describe('Skeleton test', () => {
  it('Should display element with class animate-shimmer when render is correctly', () => {
    render(<Skeleton />);
    const skeleton = screen.getByTestId('skeleton');
    expect(skeleton).toHaveClass('animate-shimmer');
  });
});
