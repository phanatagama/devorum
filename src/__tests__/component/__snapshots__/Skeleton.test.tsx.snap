// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton test Should render correctly 1`] = `
<div>
  <div
    class="animate-shimmer bg-[#f6f7f8]"
    style="background-size: 700px 100%; background-repeat: no-repeat;"
  />
</div>
`;
